pepper_cam_topic: /pepper_robot/camera/front/camera/image_raw
input_image_topic: /testcam/image_raw
detection_topic: /pepper_object_detection/detection